Tools/jhbuild/jhbuild-minimal.modules

```diff
@@ -156,6 +156,37 @@
             checkoutdir="libxml2-${version}"/>
   </autotools>
 
+  <!-- Allows USE_AVIF support -->
+  <cmake id="libavif" cmakeargs="-DCMAKE_BUILD_TYPE=Release -DAVIF_ENABLE_WERROR=OFF">
+    <pkg-config>libavif.pc</pkg-config>
+    <dependencies>
+      <dep package="aom"/>
+      <dep package="libyuv"/>
+    </dependencies>
+    <branch module="AOMediaCodec/libavif"
+            repo="github.com"
+            version="0.9.3"
+            tag="v0.9.3"/>
+  </cmake>
+
+  <cmake id="libyuv">
+    <branch module="libyuv/libyuv"
+            repo="chromium.googlesource.com"
+            tag="stable"
+            checkoutdir="libyuv"/>
+  </cmake>
+
+  <cmake id="aom" cmakeargs="-DBUILD_SHARED_LIBS=1 -DENABLE_DOCS=OFF -DENABLE_EXAMPLES=OFF -DENABLE_TOOLS=OFF -DENABLE_TESTS=OFF">
```

Comment:
please note that aom is also defined at Tools/gstreamer/jhbuild.modules

Reply:
Yes, I'm aware of that. I don't see other way but to include `aom` in the jhbuild-minimal as well, since it should be possible to build using the minimal moduleset without relying on `gstreamer.modules`.